src/test/java/uk/gov/hmcts/reform/orgrolemapping/helper/TestDataBuilder.java

```diff
@@ -490,6 +517,33 @@ public static Appointment buildAppointmentWithParams(String epimms, String isPri
                 .build();
     }
 
+    public static AppointmentV2 buildAppointmentWithParamsV2(String epimms, String isPrinciple, String appointment,
+                                                         String appointmentType, LocalDate startDate, LocalDate endDate,
+                                                         List<String> roles, String serviceCode) {
+        List<String> serviceCodes;
+        if (serviceCode != null) {
+            serviceCodes = List.of(serviceCode);
+        } else {
+            serviceCodes = null;
+        }
+
+        return AppointmentV2.builder()
+                .baseLocationId("827")
+                .epimmsId(epimms)
+                //.courtName("Fairfax County Courthouse")
+                .cftRegionID("NULL")
+                .cftRegion("South East")
+                //.locationId("1")
+                //.location("South East")
+                .isPrincipalAppointment(isPrinciple)
+                .appointment(appointment)
+                .appointmentType(appointmentType)
+                .startDate(startDate)
+                .endDate(endDate)
+                //.roles(roles)
```

Comment:
Can we update to reflect V2 formats.  e.g.
* remove `courtName`
* populate `cftRegionID`
* remove `locationId` & `locationId`
* remove `roles` **including from input params**

Reply:
Recommended changes made